app/modules/net.c

```diff
@@ -1432,6 +1434,203 @@ static int net_multicastLeave( lua_State* L )
 	return net_multicastJoinLeave(L,0);
 }
 
+// Returns NULL on success, error message otherwise
+static const char *append_pem_blob(const char *pem, const char *type, uint8_t **buffer_p, uint8_t *buffer_limit, const char *name) {
+  char unb64[256];
+  memset(unb64, 0xff, sizeof(unb64));
+  int i;
+  for (i = 0; i < 64; i++) {
+    unb64["ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/"[i]] = i;
+  }
+
```

Comment:
Yes -- I started out with that module, but it throws lua errors on invalid input (which would cause a memory leak in my layer). Also, the PEM file has embedded newlines within it that would cause problems. Originally I constructed a single base 64 string and passed that over, but it is simpler to just do the conversion....